### ISSUE TITLE
Energy reporting in kWh

### ISSUE THREAD

**Issue by rajil** [openshwprojects/OpenBK7231T_App#821]
Hello,
Both Tasmota and Shelly device report energy in kWh. Can we get Openbeken to also report in kWh instead of Wh.

Thanks

**Comment by openshwprojects** [comment]
Good idea, I can add a flag for that. I can't change it globally (but I could make this flag on by default for new builds) because many users already have it configured in Wh.

**Comment by rajil** [comment]
Yes, that would work.

**Comment by openshwprojects** [comment]
Ok, I am working on, will you be able to check if it works in few hours?

**Comment by openshwprojects** [comment]
The new kWh flag, it should be added in latest build already.

**Comment by rajil** [comment]
Thanks, it worked

**Comment by rajil** [comment]
mqtt is still reporting in Wh instead of kWh.

**Comment by rajil** [comment]
The sensor are still displaying in Wh in Home Assistant

![image](https://user-images.githubusercontent.com/379266/236636448-222853a9-6ac7-4e4f-a49a-b3b03dd3b9bf.png)


This might be the reason  i am still not able to combine energy of Tasmota and Openbeken devices.

**Comment by openshwprojects** [comment]
Have you done again the HASS Discovery?

**Comment by rajil** [comment]
Thanks, after HASS discovery it is showing in kWh.
